charts/telegraf-ds/templates/role.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.rbac.create }}
+# Allow to define extra custom rules
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: influx:telegraf:custom
+  labels:
+    rbac.authorization.k8s.io/aggregate-view-telegraf-custom: "true"
+rules:
```

Comment:
Because in my tests, I also included kube_inventory which relies on ClusterRoleAggregation. So I unified this. As David told me kube_inventory is more adapted for telegraf chart instead of telegraf-ds, I remove kube_inventory but kept role this way.

Please define what you expect, I'll update accodingly.